views/auth/temp-password.ejs

```diff
@@ -0,0 +1,47 @@
+<%- include('../layouts/header.ejs'); %>
+<div class="box">
+    <div>
+        <p>Your Temporary Password</p> 
+    </div>
+    <div>
+        <input type="text" class="password" id="temp-password" placeholder="Create Your Password" readonly="">
+        <i class="fa fa-copy copy-clipboard" onclick="copyPassword()"></i>
+    </div>
+    <p id="show-copied">Copied</p>
+    <div class="btn-container">
+        <button class="btn" onclick="randomPassword()">
+            Get Your Password
+        </button>
+    </div>
+</div>
+<script type='text/javascript'>
+    var counter = 0;
+    function randomPassword(){
+        var chars = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!@#$%^&*_+-?";
+        var passwordLength = 14;
+        var password = "";
+        
+        if (counter < 1)
+        {
+            for (var i = 0; i < passwordLength; i++) {
+                var randomNum = Math.floor(Math.random() * chars.length);
+                password += chars.substring(randomNum, randomNum+1);
+            }
+            document.getElementById("temp-password").value = password;
+            counter++;
+        }
+    }
+
+    function copyPassword() {
+        let copyPassword = document.getElementById("temp-password");
+        copyPassword.select();
+        copyPassword.setSelectionRange(0, 99999);
+        navigator.clipboard.writeText(copyPassword.value);
+        let show = document.getElementById("show-copied");
+
+        if ((show.style.display != "none")&&(counter == 1)) {
+            show.style.display = "inline-block";
+        }
+
```

Comment:
Create a JS folder in assets folder and then create a js file containing these functions